bindings/generator/generate.py

```diff
@@ -446,6 +488,13 @@ class ParsingPlaceholder:
             placeholder.__dict__ = struct.__dict__
             placeholder.__class__ = struct.__class__
             structs.append(placeholder)  # type: ignore[arg-type]
+        elif isclass(item) and issubclass(item, Enum):
+            placeholder = TYPES_DB[item]
+            annotations = getattr(item, "__annotations__", {})
+            enum = EnumInUse(member_names=item._member_names_, name=item.__name__)
+            placeholder.__dict__ = enum.__dict__
+            placeholder.__class__ = enum.__class__
+            enums.append(placeholder)
```

Comment:
should add a `# type: ignore[arg-type]` here

Reply:
done